src/badge.jsx

```diff
@@ -0,0 +1,119 @@
+const React = require('react');
+const Typography = require('./styles/typography');
+const DefaultRawTheme = require('./styles/raw-themes/light-raw-theme');
+const ThemeManager = require('./styles/theme-manager');
+
+// Badge
+export default React.createClass({
+  contextTypes: {
+    muiTheme: React.PropTypes.object,
+  },
+  //for passing default theme context to children
+  childContextTypes: {
+    muiTheme: React.PropTypes.object,
+  },
+  getChildContext () {
+    return {
+      muiTheme: this.state.muiTheme,
+    };
+  },
+  propTypes: {
+    className: React.PropTypes.string,
+    primary: React.PropTypes.bool,
+    secondary: React.PropTypes.bool,
+    primaryText: React.PropTypes.string.isRequired,
+    color: React.PropTypes.string,
+    backgroundColor: React.PropTypes.string,
+    style: React.PropTypes.object,
```

Comment:
37cbc1d5d7ee4a90bffc0960f67ad088cd3f9815

I wasn't sure if the prefixer covers similar properties or just prefixed properties with the same name, so I've left in the prefixed similar properties for now.